packages/cashier/src/Containers/withdrawal.jsx

```diff
@@ -69,18 +75,24 @@ const Withdrawal = ({
     }, [check10kLimit]);
 
     React.useEffect(() => {
-        if ((iframe_url || verification_code) && isDesktop()) {
+        if (verification_code && isDesktop()) {
             if (isCryptocurrency(currency) && typeof setSideNotes === 'function') {
-                const side_notes = [
-                    <WithdrawalSideNote key={0} />,
-                    ...(/^(UST)$/i.test(currency) ? [<USDTSideNote type='usdt' key={1} />] : []),
-                    ...(/^(eUSDT)$/i.test(currency) ? [<USDTSideNote type='eusdt' key={1} />] : []),
-                ];
+                const side_notes = [];
+                if (crypto_transactions.lenght) {
+                    side_notes.push(<RecentTransaction key={2} />);
+                } else {
```

Comment:
Don't we need both `RecentTransaction` and `WithdrawalSideNote` according to the design?

Reply:
Yes, we do